### ISSUE TITLE
<podcast:value> default splits for podcast apps?

### ISSUE THREAD

**Issue by jamescridland** [Podcastindex-org/podcast-namespace#134]
[This thread](https://podcastindex.social/web/statuses/105288396879442066) poses the question...

"If someone listens to (ExamplePodcast) on (PodcastApp). Then what's the natural split?"

I'm quite excited about the possibility of podcast apps being able to share in any micropayments from podcasting. That allows for significant possibilities for podcast apps to spend time to help improving their products.

It's fair that a podcast app should benefit from revenue being earned, just as MasterCard benefits each time I use their card. So - as a podcaster, I'd like to be able to donate, say, 3% of revenue to the podcast app, whoever that is.

Is there anything in suggesting that a default is 3% of all revenue is paid to the podcast app, and that podcast apps may adjust the <podcast:value> split to 3% if there is no podcast app split? But to honour the app split if it's been added to the podcast feed?

So if I've said:

`value:!mypaymentaddress`

then the podcast app *can* amend this to read

`value:!mypaymentaddress split:97%; value:!podcastapppaymentaddress split:3%`

BUT if the podcaster has decided a split for the app - might be bigger, might be smaller - then it isn't allowed be changed by the podcast app?

And how might we as podcasters be able to signal that whichever podcast app this is playing on can take 3%?

This is a very badly written post, I'm sorry. But now that the discussion has moved away from crypto-funny-money into proper dollars and cents, I understand it more and am beginning to consider the opportunities.

**Comment by adamc199** [comment]
I disagree that any comparison can be made to MasterCard. 

This is about value. 
Perhaps apps should tell the listener what their suggested split is. 

We have always suggested:
1% app
1% index (not in your example)
98% podcaster, who can sub divide at will 
This leaves room for podcaster and app developer to work out promotional agreements for a larger split. 

Remember that this is based on a pledge from the listener. This should be transparent to the person pledging. 

Comparing this to traditional payments and infrastructure simply isn’t appropriate when you look at a the possible combinations and features yet to be developed.

**Comment by jamescridland** [comment]
OK - let's not get bogged down in the 1% vs 3%. Let's go with 1%.

What happens if I, as a podcaster, want to actually give app developers 5% instead? Can I do that?

If a podcaster simply doesn't include an "app" or "index", does that mean the app/index never gets paid?

**Comment by adamc199** [comment]
The value block is entirely voluntary and completely controlled by the feed owner. 
No index in the block, no payment. 

I’m going to propose these is a special entry in the value block that is “app” or “player” with a split percentage. The podcast app will use that number to assign their nodeID

So technically a podcaster could  assign any percentage for the app or the index for that matter, as long as the total is 100%

This will create interesting markets for podcasters with a high “app” percentage as I can imagine that may result in beneficial placement or promotion. 

Transparency will be key.

**Comment by daveajones** [comment]
With regards to transparency, we need to surface this on the PI website.  When a feed contains a value block (lightning or other) we can show what the splits are on the PI podcast page for that feed.  I'd like to show that for apps and platforms/indexes/apis also.

@jamescridland I'm thinking that the default 98/1/1 breakdown can be put into the wording of the tag in the namespace.  But, just make it clear that this is a starting assumption based on common practice, and that people should expect competition from various parties within the chain.  Maybe also some wording to app developers to encourage them to make their split well-known and public.

**Comment by daveajones** [comment]
And also encourage app developers to put UI (sliders, incrementer boxes, etc) into their apps to allow modifying the splits for different podcasts they listen to.  There is already "per podcast" settings for things like playback speed, so keeping the custom splits makes sense.

**Comment by jamescridland** [comment]
"Maybe also some wording to app developers to encourage them to make their split well-known and public."

Who's in charge of the split? The app developer? So, if I'm on AlphaApp and BetaApp and BetaApp decides to change their split to 25%, how does this work - do I, as podcast producer, just have to accept that?

As podcast producer, if I like AlphaApp, can I elect to give them 10% but to give BetaApp, who I don't like, nothing?

**Comment by adamc199** [comment]
The podcaster determines the split in their feed. Of course an app could be a bad actor and override this. But frankly, so could the index or even the hosting company. 

Try o answer your question; yes, you could remove the generic “app” split and add a specific split for an app. 

But I’m the current state, this api would get that payment even if the show plays in a different app. 

We have an implementation that is not only working, but growing. We are learning a lot.

**Comment by brianoflondon** [comment]
And YouTube takes 30% of all pledges during live streaming! Not to mention all the various credit card and payment processing fess. 

We're building (back) better, and the only thing we can all do is push for transparency and report bad behavior. 

Crypto should teach many that they have to take more responsibility for their own actions. That's the flip side of freedom.

**Comment by daveajones** [comment]
I guess public shaming goes a long way on this. Maybe that sounds simplistic. But, any app that goes rogue and jacks things way out of proportion may land on Podnews with a “shoutout” they don’t want.

**Comment by jamescridland** [comment]
Why wouldn't we say:

**If you don't set an "app" share, then the default is 1%. Set it at 0 to disable this.**

Wouldn't that drive adoption?

**Comment by daveajones** [comment]
Yeah I like that.

**Comment by brianoflondon** [comment]
Basics:

At the moment we envisage the Podcast Playing App [the app] is going to be making the actual micro payments either direct from the listener's device or (perhaps) centrally.

The app will have have to look at the Value block in either the channel or the episode and do the maths to work out the split.

The app will always have the power to secretly or openly take a cut and/or play games.

I'd like to see a standard set that apps have an option to display where they're sending payments.

Crypto addresses like Lightning are problematic for transparency, for that reason I think we might need to make a change and add a "role" sub tag (for say Podcast or Host or App) and use the name for a Human readable name sub tag (what do you call these?)

**Comment by adamc199** [comment]
>>>>>The app will always have the power to secretly or openly take a cut and/or play games.<<<<<

Yes, just as listeners don’t have to find their wallet at all and receive “free” content. 

It’s an honor based system. 

>>>>I'd like to see a standard set that apps have an option to display where they're sending payments.<<<

Would this be used to clarify the value 4 value concept? How would this stop “games” being played?

Can you provide an example of this “set”?

**Comment by brianoflondon** [comment]
Just as an example, over on Hive where the internal inflation is paid out as rewards for posting, various apps take cuts. For example the 3speak video site takes 11% of rewards which I (and the others who use their site) feel is reasonable for the sheer horror of storing and serving video. If you hover your mouse over a post that was sent via 3speak (even on another Hive front end) this is clear:

![image](https://user-images.githubusercontent.com/28054569/101376805-bc0ce900-38b9-11eb-95a0-ca61c33722e0.png)

At this stage in development I think we need to set an expectation that apps will always surface to the users, who is getting paid which is another reason I'd very much like to see human readable names associated with crypto addresses wherever necessary. 

Footnote: BTW there is also a direct tipping function on Hive where viewers can send funds direct - the 11% example above wouldn't apply there. Hive is a very different case from this value for value model as the funds being distributed are a bit of a mirage and rabbit hole not appropriate for here.

**Comment by adamc199** [comment]
Agree on app transparency. A lightning enabled podcast app is a wallet. A wallet must surface its log of payments. The user can check in a variety of ways if this the expected distribution.

**Comment by genen** [comment]
So 3 months later when there are now 3 apps streaming sats... The index is still 1%, the podcast app varies from 1-4% (not up to podcaster) and the incoming wallet is 1-5%.  So right now it's much more expensive than just doing a Paypal donation. With sufficient competition, there may be pressure to lower fees, but right now I expect fees to go up while everyone is testing as everyone doing dev is planning on taking their share.   Not sure there is anything the index can do ultimately other than 'suggesting' a split.

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.
